FiveCalls/FiveCalls/Appearance.swift

```diff
@@ -12,10 +12,15 @@ enum Appearance {
     static func setup() {
         let pageControlAppearance = UIPageControl.appearance()
         pageControlAppearance.pageIndicatorTintColor = R.color.lightBlue()
-        pageControlAppearance.currentPageIndicatorTintColor = R.color.darkBlue()
+        pageControlAppearance.currentPageIndicatorTintColor = R.color.fivecallsDarkBlue()
```

Comment:
This doesn't seem to be working for me in SwiftUI branch, page indicators are missing.

Reply:
Probably ok for now, added a bug to #331